layouts/default.vue

```diff
@@ -715,7 +715,7 @@ export default {
           .then(() => {})
```

Comment:
Some kind of acknowledgement would suit here. Say, logging to the console or maybe a toast.

Reply:
Also, it seems there're couple of best practices while making use of the `Web Share API`. Please refer this [blog post](https://alligator.io/js/web-share-api/)

Reply:
We don't have `canonical` URL for home page.
And regarding the "logging  or showing toast": well there's no need to notify the user on successful share event. I never find it useful.